src/settings/index.ts

```diff
@@ -18,6 +18,8 @@ import defaultMapIcon from "assets/icons/default-map-icon.png";
 import sparkleNavLogo from "assets/icons/sparkle-nav-logo.png";
 import sparkleverseLogo from "assets/images/sparkleverse-logo.png";
 
+export * from "./url-settings";
```

Comment:
Can you change this file name to follow our current convention of `camelCase`

Reply:
renamed @ f908a1489cfee5b6d593c45708b24e78493e99b5